### PR TITLE
Tweak: Add `<h1>` tag to the Elementor Editor page [ED-10404]

### PR DESCRIPTION
**Pull request by rami-elementor**
Related: #22015

### CODE REVIEW

**Review by rodolphebertozzo — DISMISSED** [review]

File: core/editor/templates/editor-body-v1.view.php [hunk 1]
```diff
@@ -10,6 +10,8 @@
 $notice = Plugin::$instance->editor->notice_bar->get_notice();
 ?>
 
+<h1 class="elementor-screen-only"><?php echo esc_html__( 'Elementor Editor', 'elementor' ); ?></h1>
```
Comment (arielk) [inline core/editor/templates/editor-body-v1.view.php:13]: Please change the title to: `Edit <$post_title> - Elementor`
Use `$post_id` as a fallback if there is no title.

File: core/editor/templates/editor-body-v2.view.php [hunk 2]
```diff
@@ -10,6 +10,8 @@
 $notice = Plugin::$instance->editor->notice_bar->get_notice();
 ?>
 
+<h1 class="elementor-screen-only"><?php echo esc_html__( 'Elementor Editor', 'elementor' ); ?></h1>
```
Comment (arielk) [inline core/editor/templates/editor-body-v2.view.php:13]: As above

File: tests/qunit/index.html [hunk 3]
```diff
@@ -6,6 +6,8 @@
 	<span>{{{  }}}</span>
 </script>
 
+<h1 class="elementor-screen-only">Elementor Editor</h1>
```
Comment (arielk) [inline tests/qunit/index.html:9]: Same

**Review by arielk — CHANGES_REQUESTED** [review]

File: core/editor/templates/editor-body-v1.view.php (hunk 1 above)
Comment (rami-elementor) [reply core/editor/templates/editor-body-v1.view.php:13]: No problem, I will add the post title, but I think **Edit "%s" with Elementor** title makes much more sense.

**Review by arielk — APPROVED** [review]

**Pull request merged**